pkg/debug/scripts/collect-info.sh

```diff
@@ -44,18 +47,17 @@ DIR="$TMP_DIR/$INFO_DIR"
 mkdir -p "$DIR"
 mkdir -p "$DIR/network"
 
-# Install GNU version of the 'ps', 'tar' and other tools
-echo "- install GNU tools"
-apk add procps tar dmidecode >/dev/null 2>&1
+# Check for internet access, timeouts at 30 seconds,
+# If there is internet access, install required GNU utilities and networking tools.
+nc -z -w 30 dl-cdn.alpinelinux.org 443 2>/dev/null
+if [ $? -eq 0 ]; then
```

Comment:
I shellcheck thing to fix here. FWIW you can install and run shellcheck on your laptop.